docs/config.md

```diff
@@ -22,7 +27,24 @@ RUN cp $PHP_INI_DIR/php.ini-production $PHP_INI_DIR/php.ini
 RUN cp $PHP_INI_DIR/php.ini-development $PHP_INI_DIR/php.ini
 ```
 
-If you don't use Docker, copy one of `php.ini-production` or `php.ini-development` provided [in the PHP sources](https://github.com/php/php-src/).
+dev.Dockerfile:
```

Comment:
You can drop this section, or move it to `CONTRIBUTING.md`, `dev.Dockerfile` is only for contributors to FrankenPHP itself.